### PR TITLE
Add ADU + ratings to the StaticAddonCard

### LINKED ISSUE #10338: Show number of users and ratings in `StaticAddonCard`
This issue is about adding the number of users and ratings to the `StaticAddonCard` for the blog.

See:

Screen Shot 2021-04-08 at 18 53 05

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #10338

---

Note: some of the style has been updated to look more like the latest mocks. This isn't strictly compliant with the mocks but it is consistent with the AMO frontend.

Screen Shot 2021-04-12 at 10 48 21

Screen Shot 2021-04-13 at 10 49 04

### CODE REVIEW

File: src/blog-utils/StaticAddonCard/index.js [hunk 1]
```diff
@@ -8,20 +8,30 @@ import AddonTitle from 'amo/components/AddonTitle';
...
 } from 'amo/components/GetFirefoxButton';
+import Rating from 'amo/components/Rating';
+import translate from 'amo/i18n/translate';
 import type { AddonType } from 'amo/types/addons';
+import type { I18nType } from 'amo/types/i18n';
 
 import './styles.scss';
 
-type Props = {|
+type PublicProps = {|
```
Comment (bobsilverberg) [inline src/blog-utils/StaticAddonCard/index.js:18]: Everywhere else we use `Props` to mean "public props" and `InternalProps` to mean "all of the props", but here you've reversed that and introduced `PublicProps`. Can we make this consistent with the rest of the code in the repo?

File: src/blog-utils/StaticAddonCard/index.js [hunk 2]
```diff
@@ -35,19 +45,37 @@ const StaticAddonCard = ({ addon }: Props): React.Node => {
...
+      </div>
+
+      <div className="AddonMetadata">
+        {averageRating && (
+          <Rating rating={averageRating} readOnly styleSize="small" />
+        )}
+
+        {averageDailyUsers && (
+          <p className="AddonMetadata-adu">
+            Users: {i18n.formatNumber(averageDailyUsers)}
```
Comment (bobsilverberg) [inline src/blog-utils/StaticAddonCard/index.js:62]: Why are we bothering to localize the format of the number when we're not even translating the string "Users:"?

File: src/blog-utils/StaticAddonCard/index.js [hunk 3]
```diff
@@ -35,19 +45,37 @@ const StaticAddonCard = ({ addon }: Props): React.Node => {
...
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={sanitizeHTML(nl2br(summary), ['a', 'br'])}
-      />
-
-      <GetFirefoxButton
-        addon={addon}
-        buttonType={GET_FIREFOX_BUTTON_TYPE_ADDON}
-        className="AddonFirefoxButton"
-      />
+      <div className="AddonSummary">
```
Comment (bobsilverberg) [inline src/blog-utils/StaticAddonCard/index.js:48]: Are we not sticking with our usual convention for classNames for this component (e.g., `StaticAddonCard-AddonSummary`)?

**Review by bobsilverberg — APPROVED** [review]
Looks good. A few questions, particularly about conventions.

r+wc

File: src/blog-utils/StaticAddonCard/index.js (hunk 1 above)
Comment (willdurand) [reply src/blog-utils/StaticAddonCard/index.js:18]: sure

File: src/blog-utils/StaticAddonCard/index.js (hunk 2 above)
Comment (willdurand) [reply src/blog-utils/StaticAddonCard/index.js:62]: because the number must be formatted.

File: src/blog-utils/StaticAddonCard/index.js (hunk 3 above)
Comment (willdurand) [reply src/blog-utils/StaticAddonCard/index.js:48]: we should 😉 I'll update the class names.

**Pull request merged**